torch/csrc/autograd/saved_variable.cpp

```diff
@@ -37,28 +37,34 @@ SavedVariable::SavedVariable(const Variable& variable, bool is_output, bool is_i
       "you can make a clone to get a normal tensor and use it in autograd.")
 
     was_default_constructed_ = false;
+    is_inplace_view_ = is_inplace_view;
```

Comment:
nit: following the discussion we had, could you rename this in `is_inplace_on_view`? to reflect that we're not talking about inplace view operations but inplace operations performed on a view?